[BodyPix] - increase max internal resolution to 2

oveddan opened the pull request:
For BodyPix - increase max internal resolution to 2, so that inference can be run to emulate a lower output stride and get higher accuracy.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/tensorflow/tfjs-models/357)
<!-- Reviewable:end -->

nsthorat commented on body-pix/src/util.ts:
```diff
@@ -71,10 +74,11 @@ function toInternalResolutionPercentage(
+            internalResolution >= MIN_INTERNAL_RESOLUTION,
         () =>
-            `inputResolution must be a string or number between 0 and 1, but ` +
+            `inputResolution must be a string or number between 0 and 4, but ` +
```
this doesn't match the constant -- and let's use the constant in the error instead of hardcoding the string.

oveddan commented:
___
*[body-pix/src/util.ts, line 81 at r2](https://reviewable.io/reviews/tensorflow/tfjs-models/357#-LtjXl4pNASQXXcpPg2T-r2-81:-LvCMzXxCd_87cFwD0mi:bz8nf0i) ([raw file](https://github.com/tensorflow/tfjs-models/blob/9da13d5d88c7a76fe1809513c519399d916f869e/body-pix/src/util.ts#L81)):*
<details><summary><i>Previously, nsthorat (Nikhil Thorat) wrote…</i></summary><blockquote>

this doesn't match the constant -- and let's use the constant in the error instead of hardcoding the string.
</blockquote></details>

Good call.  Fixed


<!-- Sent from Reviewable.io -->

oveddan commented:
___
*[body-pix/src/util.ts, line 81 at r2](https://reviewable.io/reviews/tensorflow/tfjs-models/357#-LtjXl4pNASQXXcpPg2T-r2-81:-LvCNIa17fhe5ogCWYj4:bdmf86b) ([raw file](https://github.com/tensorflow/tfjs-models/blob/9da13d5d88c7a76fe1809513c519399d916f869e/body-pix/src/util.ts#L81)):*
<details><summary><i>Previously, oveddan (Dan Oved) wrote…</i></summary><blockquote>

Good call.  Fixed
</blockquote></details>

Fixed in PR #365  


<!-- Sent from Reviewable.io -->
